### DISCUSSION TITLE
Checker for vendor/bin/php-cs-fixer

### DISCUSSION THREAD

**Discussion by alexeygeno** [PHP-CS-Fixer/PHP-CS-Fixer#7301] [Feature ideas]
Hello, guys. Thank you for the project.
I didn't find in the docs something like:
```shell
vendor/bin/php-cs-fixer check
```
Sometimes(in CI for example) it is convenient to check if the code style is ok not trying to fix it. 
How about implementing it?

**Comment by Wirone** [comment]
It's already implemented via `fix --dry-run` 😉. But maybe dedicated entrypoint wouldn't be a bad idea... 🤔.

**Reply by keradus** [reply]
I think it's OK, as long as we claim it's only the alias and not reimplement the command :D

**Reply by Wirone** [reply]
Created [an issue](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7307).

**Comment by alexeygeno** [comment]
Thank you for the quick reply 😉.  
`fix --dry-run` should work well for CI. As I see it returns a non-zero exit status if there are files to fix.

**Reply by Wirone** [reply]
Yes, it fails the job if there are violations, we use it internally for our own pipelines 🙂.

**Reply by alexeygeno** [reply]
Great🙂. I think the 'check' alias could be added, at least to be included on the main project page in [the usage paragraph](https://cs.symfony.com/#usage). This way, developers will see right away that the project has a 'checker' as well. But it's just my opinion 😉.
